test/mobile/nnc/test_nnc_backend.cpp

```diff
@@ -187,49 +95,6 @@ TEST(NNCBackendTest, AOTCompileThenExecute) {
   EXPECT_TRUE(result.toTensor().equal(reference.toTensor()));
 }
 
-TEST(NNCBackendTest, FakeTensor) {
```

Comment:
I had to remove this test as we won't be able to compile models with calls to methods of custom classes (at least as of now). The other test still exercises the AOT workflow.